.github/workflows/remove-offline-installer.yml

```
@@ -0,0 +1,45 @@
+name: Remove Tagged Offline Installer
+
+on:
+  workflow_dispatch:
```

Adding a [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) to this workflow might fit. If the workflow wouldn't require any `inputs`. Like listing all the releases and removing x, y, z depending on some criteria. 
```suggestion
on:
  schedule:
    - cron:  '30 5,17 * * *'
  workflow_dispatch:
```